port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.port/spec.yaml

```diff
@@ -1,14 +1,14 @@
 description: {{cookiecutter.integration_name}} integration for Port Ocean
-icon: Cookiecutter # Should be one of the available icons in Port
+icon: Blueprint # Should be one of the available icons in Port
 features:
   - type: exporter
     section: Git Providers # Should be one of the available sections in Port
     resources:
-      - kind: <ResourceName1>
-      - kind: <ResourceName2>
+      - kind: {{ cookiecutter.integration_slug }}-example-kind
+      # - kind: <ResourceName2>
```

Comment:
remove?

Reply:
I wanted to add that as an example of what could be added